### ISSUE TITLE
Fretboard diagrams are not exported to MusicXML

### ISSUE THREAD

**Issue by Jojo-Schmitz** [musescore/MuseScore#16902]
### Issue type

Other type of issue

### Bug description

See https://musescore.org/en/node/303623

### Steps to reproduce

1) Create new Musescore file.
2) Select instruments.
3) Add one acoustic guitar stave and one linked tab stave.
4) Using the advanced palette, add any chord diagram.
5) Export the file to musicxml.
6) No chord diagram is in the file (I tried visualizing the file using Soundslice's mxl viewer and manually inspecting the xml content).

### MuseScore Version

4.0.2

### Regression

Yes, this used to work in Musescore 3.6 and now is broken

### Operating system

Windows 11

### Additional context

Basically this is because https://github.com/musescore/MuseScore/pull/5958 never got merged or https://github.com/musescore/MuseScore/pull/7167 never got ported over to master, so it got fixed only for 3.6

**Comment by Jojo-Schmitz** [comment]
Guess you can assign it to me ;-)
